bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -79,6 +79,36 @@ void CreateInstance(bigtable::InstanceAdmin instance_admin, int argc,
 }
 //! [create instance]
 
+//! [update instance]
+void UpdateInstance(bigtable::InstanceAdmin instance_admin, int argc,
+                    char* argv[]) {
+  if (argc != 2) {
+    throw Usage{"update-instance: <project-id> <instance-id>"};
+  }
+  std::string instance_id = ConsumeArg(argc, argv);
+  auto instance = instance_admin.GetInstance(instance_id);
+  // Modify the instance and prepare the mask with modified field
+  instance.set_display_name("Modified Display Name");
+  std::string mask_text = R"(
+  paths: 'display_name'
+  )";
+  google::protobuf::FieldMask mask;
+  google::protobuf::TextFormat::ParseFromString(mask_text, &mask);
+  auto future = instance_admin.UpdateInstance(&instance, &mask);
+  // Most applications would simply call future.get(), here we show how to
```

Comment:
Feel free to simplify this sample to just call `future.get()`, one of the samples should show the general case, not all of them need to.

Reply:
fixed